ani-cli

```diff
@@ -83,12 +85,24 @@ update_script() {
 # checks if dependencies are present
 dep_ch() {
 	for dep; do
-		if ! command -v "$dep" >/dev/null; then
-			die "Program \"$dep\" not found. Please install it."
-		fi
+        if ! command -v "$dep" >/dev/null; then
+            if [ "$dep" = "flatpak run io.mpv.Mpv" ]; then
+                if flatpak info io.mpv.Mpv 2>&1 | grep -q 'error'; then
```

Comment:
Can't this be expressed with `||` or `&&`?

Reply:
It probably can. Ill work on it.

Reply:
i came up with this
```bash
dep_ch() {
    for dep; do
        if ! ( command -v "$dep" >/dev/null ) && ! [ "$dep" = "flatpak run io.mpv.Mpv" ]; then
            die "Program \"$dep\" not found. Please install it."
        fi
        if flatpak info io.mpv.Mpv 2>&1 | grep -q 'error'; then
            die "Program \"$dep\" not found. Please install it."
        fi
      done
 }
 ```
 
 but i dont like it because it checks for flatpak mpv even when flatpak is not installed

Reply:
I'll accept it as-is, we're working on a better version in the v4 branch anyway. The best would be to revert it completely and maybe put a todo comment at the function definition

Reply:
what should i do about exiting when flatpak is selected?